ibm/resource_ibm_is_instance.go

```diff
@@ -1535,39 +1571,6 @@ func instanceGet(d *schema.ResourceData, meta interface{}, id string) error {
 		d.Set(isInstancePrimaryNetworkInterface, primaryNicList)
 	}
 
-	if instance.NetworkInterfaces != nil {
```

Comment:
Why is this code removed from READ method?

Reply:
in instance reource, network interface is an optional field and NOT computed. so when we create a networkintf with the new networkintf resource, in the next apply,  the READ method of instance resource reads all the networkintf and sets the values. and in next apply, as it is the field is optional only, it detects a change. So, i had a discussion with @hkantare and decided we will remove it from read method and add it in the import  logic